tutorials/chapter1.md

```diff
@@ -342,8 +379,7 @@ private:
     kpsr::Publisher<float> * _publisher;
 
     float calculateModule(const std::vector<float> & event) {
-      // TODO: use lambda to calculate the module
-      return 0f;
+       return sqrt(std::inner_product(event.begin(), event.end(),event.begin(), 0.0f));
```

Comment:
please respect the indentation. It looks better if all is uniformly indented. Also PRs are harder to review with indentation changes.